src/riot-rs-embassy/src/arch/nrf52.rs

```diff
@@ -59,9 +59,30 @@ pub mod usb {
 
 #[cfg(feature = "hwrng")]
 pub mod hwrng {
+    use crate::arch;
+
     embassy_nrf::bind_interrupts!(pub struct Irqs {
```

Comment:
https://github.com/future-proof-iot/RIOT-rs/pull/257#discussion_r1565876349 applies here (was marked as resolved but is still pub here)